mlem/utils/module.py

```diff
@@ -266,8 +266,10 @@ def get_package_name(mod: ModuleType) -> str:
     """
     if mod is None:
         raise ValueError("mod must not be None")
-    name = mod.__name__
-    return MODULE_PACKAGE_MAPPING.get(name, name)
+    if isinstance(mod, str):
+        mod = importing.import_module(mod)
+    # QUESTION: should we issue a warning if this list have > 1 item?
```

Comment:
My take is to print a warning for now, but need to investigate in future to resolve
https://github.com/iterative/mlem/issues/524#issuecomment-1346420956

Reply:
I think you should print a warning of course, but if you print it you also should provide a way to resolve it. Eg manually configuring the mapping